index.js

```diff
@@ -89,7 +89,7 @@ client.on("message", message => {
 		command.execute(message, args);
 	} catch (error) {
 		console.error(error);
-		message.reply("Unexpected error.");
+		message.reply(`unexpected error! \n${error}`);
```

Comment:
I know że wywalanie errora w discordzie byłoby wygodne dla nas ale robiłoby to straszny spam i też randomowych userów nie obchodzi (i nie powinno) jakie errory nam wyskakują

Reply:
Teoretycznie prawda, dodałem to tymczasowo, bo jestem leniwy i nie chce mi się Alt+Tab do konsoli. Pozdrawiam XD

Reply:
Poza tym, jedynie kto korzysta z tego bota na razie to te same osoby, które piszą, więc mogłoby być póki co XD

Reply:
@lufv no a potem co z tym zrobisz? XD

Reply:
a dobra, mówisz o errorze, myślałem że o display